src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -51,6 +68,11 @@ public function __construct($propertyKey)
  */
 class DataHelper
 {
+    /*
+     * Internal character limit for attribute values.
+     */
+    const CHARACTER_LIMIT = 16383;
```

Comment:
Value is redundant - also present in exception. Maybe it can be centralized, e.g. defined here and passed to the exception on construction. This is also a good idea because in case the value changes and the check is updated, the exception could still print an outdated message then.

Reply:
Done :heavy_check_mark: